src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -12,7 +12,7 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{5471BFE8-8071-466F-838E-5ADAA779E742}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <UseECMAKey Condition="'$(UseECMAKey)'==''">true</UseECMAKey>
```

Comment:
Right, but I'm not sure but it looks like it supports it now. I tried locally and csc is able to build and sign the binary using Ecma key.